### ISSUE TITLE
Trait Suggestion: Long rays/lines S38, S39, S153 are Cozero complemented P61

### ISSUE THREAD

**Issue by Moniker1998** [pi-base/data#1047]
## Trait Suggestion

Long rays/lines S38, S39, S153 are Cozero complemented P61, but this fact is not known to pi-Base today:
[link to pi-Base 1](https://topology.pi-base.org/spaces/S000038/properties/P000061)
[link to pi-Base 2](https://topology.pi-base.org/spaces/S000039/properties/P000061)
[link to pi-Base 3](https://topology.pi-base.org/spaces/S000153/properties/P000061)

## Proof/References

Even continuous function $f:X\to \mathbb{R}$ is eventually constant. If $U\subseteq X$ is a cozero-set, then either $U$ is bounded from above, or $U$ contains all values larger than $x$ for some $x\in X$. 

In $U$ is bounded from above, say $U\subseteq \\{y\in X : y < x\\}$, then since $\\{y\in X : y < x\\}$ is homeomorphic to a subspace of $\mathbb{R}$, its [cozero complemented](https://topology.pi-base.org/spaces?q=euclidean+%2B+not+cozero+complemented), so there is open $V\subseteq \\{y\in X : y < x\\}$ disjoint from $U$ such that $U\cup V$ is dense in $\\{y\in X : y < x\\}$. If $V_0 = V\cup \\{y\in X : y > x\\}$ then $U, V_0$ are disjoint cozero sets and $U\cup V_0$ is dense in $X$.

If $U$ contains $\\{y\in X : y > x\\}$, let $U_0 = U\cap \\{y\in X : y < x\\}$ and find open $V\subseteq  \\{y\in X : y < x\\}$ disjoint from $U_0$ such that $U_0\cup V$ is dense in $\\{y\in X : y < x\\}$. Then $U, V$ are disjoint cozero sets and $U\cup V$ is dense in $X$.

**Comment by Moniker1998** [comment]
As in #1044 those spaces are LOTS. This makes me think that perhaps every LOTS is cozero complemented.

**Comment by yhx-12243** [comment]
> As in #1044 those spaces are LOTS. This makes me think that perhaps every LOTS is cozero complemented.

No. S141 (Ordered space $\omega_1 + 1 + \omega^*$) is a counterexample.

> Clearly the $U = \omega^*$ is cozero. If $X$ is cozero complemented, then $V$ is a cozero subset of $\omega_1 + 1$, thus a cozero subset of $\omega_1$ (cozero set must be open).
>
> However, cozero subset of $\omega_1$ are either bounded, are either containing the full tail. If bounded, then $U \cup V$ is not dense in $X$. If containg the full tail, then it containing the middle “1”, both into a contradiction.
